pkg/utils/kubernetes/utils.go

```diff
@@ -0,0 +1,37 @@
+package kubernetes
+
+import (
+	"errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func CustomResourceDefinitionExists(gvk schema.GroupVersionKind, cfg ...*rest.Config) (bool, error) {
+	var err error
+	if len(cfg) > 0 && cfg[0] == nil {
+		cfg = cfg[0:0]
+	}
+	if len(cfg) == 0 {
+		tcfg, err := config.GetConfig()
+		if err != nil {
+			return false, err
+		}
+		cfg = append(cfg, tcfg)
+	}
+	if err != nil {
```

Comment:
This block is not needed. Would have returned with error by this point, if there was one.

Reply:
fixed

Reply:
Same here, it's still the same

Reply:
I don't understand.
what the code is doing: L20 checks L19, L21 returns if err.

Reply:
My mistake. It's fixed, as you said, and this is a different check.